docs/_docs/Edison/Setting Up/1.2-Setting-up.md

```diff
@@ -26,9 +26,9 @@ product: Edison
 
         git checkout master
 
-You can also checkout dizzy-uptodate, dizzy-latest, dizzy-rt or morty, pyro64, pyro64-acpi, rocko32, rocko64-acpi, sumo32 and sumo64-acpi (all frozen) or thud
+You can also checkout dizzy-uptodate, dizzy-latest, dizzy-rt or morty, pyro64, pyro64-acpi, rocko32, rocko64-acpi, sumo32 and sumo64-acpi, thud (all frozen) or warrior.
 
-Master/thud is currently a continuation of sumo32 with acpi enabled by default and for thud with enabled x86_64 on top of that.
+Master/warrior is currently a continuation of sumo32 with acpi enabled by default and for  with enabled x86_64 on top of that.
```

Comment:
typo: missing word in the "and for  with enabled x86_64" piece (was thud previously)